scripts/sct_extract_metric.py

```diff
@@ -55,169 +57,212 @@ def get_parser():
+    optional.add_argument(
+        "-h",
+        "--help",
+        action="help",
+        help="Show this help message and exit."
+    )
+    optional.add_argument(
+        '-f',
+        metavar=Metavar.folder,
+        default=os.path.join("label", "atlas"),
+        help=(f"Single label file, or folder that contains WM tract labels."
+              f"Example: {os.path.join(path_sct, 'data', 'atlas')}")
+    )
+    optional.add_argument(
+        '-l',
+        metavar=Metavar.str,
+        default='',
+        help=("Label IDs to extract the metric from. Default = all labels. Separate labels with ','. To select a group " 
+              "of consecutive labels use ':'. Example: 1:3 is equivalent to 1,2,3. Maximum Likelihood (or MAP) is "
+              "computed using all tracts, but only values of the selected tracts are reported.")
+    )
+    optional.add_argument(
+        '-method',
+        metavar=Metavar.str,
+        choices=['ml', 'map', 'wa', 'bin', 'max'],
+        default=param_default.method,
+        help=("R|Method to extract metrics.\n"
+              "    - ml: maximum likelihood (only use with well-defined regions and low noise)\n"
+              "          N.B. ONLY USE THIS METHOD WITH THE WHITE MATTER ATLAS! The sum of all tracts should be 1 in " 
+              "          all voxels (the algorithm doesn't normalize the atlas).\n"
+              "    - map: maximum a posteriori. Mean priors are estimated by maximum likelihood within three clusters "
+              "           (white matter, gray matter and CSF). Tract and  noise variance are set with flag -p.\n"
+              "           N.B. ONLY USE THIS METHOD WITH THE WHITE MATTER ATLAS! The sum of all tracts should be 1 in "
+              "           all voxels (the algorithm doesn't normalize the atlas).\n"
+              "    - wa: weighted average\n"
+              "    - bin: binarize mask (threshold=0.5)\n"
+              "    - max: for each z-slice of the input data, extract the max value for each slice of the input data.")
+    )
+    optional.add_argument(
+        '-append',
+        type=int,
+        metavar=Metavar.int,
+        choices=(0, 1),
+        default=0,
+        help="Whether to append results as a new line in the output csv file instead of overwriting it. 0 = no, 1 = yes"
+    )
+    optional.add_argument(
+        '-combine',
+        type=int,
+        metavar=Metavar.int,
+        choices=(0, 1),
+        default=0,
+        help="Whether to combine multiple labels into a single estimation. 0 = no, 1 = yes"
+    )
+    optional.add_argument(
+        '-o',
+        metavar=Metavar.file,
+        default=param_default.fname_output,
+        help=("File name (including the file extension) of the output result file collecting the metric estimation "
+              "results.\n"
+              "Three file types are available: a CSV text file (extension .txt), a MS Excel file (extension .xls) and "
```

Comment:
we might want to take the opportunity to update this-- i don't think we cover xls and pickle anymore

Reply:
Yes, good eye. It looks like we use `save_as_csv` only in this script.

https://github.com/neuropoly/spinalcordtoolbox/blob/49cb4a16ade7c91533e669c8d1d2a777f5bbfe66/scripts/sct_extract_metric.py#L343